README.md

```diff
@@ -6,11 +6,43 @@
 
 ## Introduction
 
+This package contains the OpenAPI Assertion tests that used to be included in `@japa/assert`
+
+## Installation
+Install the package from the npm registry as follows:
+
+```sh
+npm i @japa/assert
```

Comment:
Should be @japa/openapi-assertions

Reply:
```suggestion
npm i @japa/openapi-assertions
```

Reply:
Good catch, fixed — I was kinda tired when I did this